concurrent-tck/pom.xml

```diff
@@ -132,11 +181,6 @@
                             <artifactId>derbyshared</artifactId>
                             <version>10.15.2.0</version>
                         </artifactItem>
-                        <dependency>
-                                <groupId>org.netbeans.tools</groupId>
-                                <artifactId>sigtest-maven-plugin</artifactId>
-                                <version>1.6</version>
-                        </dependency>
                     </artifactItems>
                     <outputDirectory>${project.build.directory}/server-dependencies</outputDirectory>
                 </configuration>
```

Comment:
To expand - using failsafe would allow us to utilise the `pre-integration-test` phase to automate the server config currently done manually (e.g. creating the JVM options)

Reply:
I agree to create a new ticket to rework concurrency runner.